src/components/Survey.tsx

```diff
@@ -179,9 +179,11 @@ const SurveyView: (Props: Props) => JSX.Element = ({
   const backgroundColor =
     theme === Themes.Dark ? Colors.LightBlack : Colors.White;
 
-  const addQueryParamScript = `window.history.pushState('', '', '?${params.join(
-    '&'
-  )}');`;
+  // Only do this if we haven't already
+  const addQueryParamScript = `if (!window.location.search) {
```

Comment:
@lankybutmacho there was in fact a real flicker, it was being caused by this JS injection causing an infinite loop of loading (so the loading indicator kept briefly showing up). I believe the change in an update to react-native-webview calling pushState would now trigger a reload of the page.

My fix is just to check if we've already appended the params and if so don't do it again